.changeset/hot-bats-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-datepicker': minor
```

Comment:
This will release `4.1.0` as latest. Please revert the changeset and publish manually to beta tag using `npm publish`

Reply:
Ok, the `next` branch seems good. I think the steps are:
1. rebase this with `next` branch
2. generate changeset
3. merge the PR to next

Unknown: How do we keep changes in `main` and `next` branches in-sync

Reply:
Let's publish it out manually just so that we can wrap it up today. The next branch can come later.